notebooks/colab_metl_predicting.ipynb

```diff
@@ -0,0 +1,2036 @@
+      "metadata": {
+        "id": "UENVWR2srLGD"
+      },
+      "source": [
+        "Depending on the model chosen, different files might be needed. This example is setup to use metl-l-2m-3d-gb1 and will need a few data for prediction. **Link to the model descriptions here again to describe the IO of each model**.\n",
+        "\n",
+        "Specifically, for this 3d gb1 model we will need:\n",
+        "- A wild type sequnece\n",
+        "- a PDB structure file (as this is a 3d model)\n",
+        "- variants to use with METL"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "HBAocw_urLGE",
+        "cellView": "form"
+      },
+      "outputs": [],
+      "source": [
+        "# @title Protein wild type\n",
+        "# @markdown Enter the wild type of your protein here. The wildtype for gb1 is provided to use with the default model example here.\n",
+        "wildtype = 'MQYKLILNGKTLKGETTTEAVDAATAEKVFKQYANDNGVDGEWTYDDATKTFTVTE' # @param {type:\"string\", placeholder:\"Enter a wildtype here\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9H12FqxWrLGF"
+      },
+      "source": [
+        "The PDB file is something that we will reference from within the colab through its file path. If you don't need a PDB file for your use-case, simply set this variable to None."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "nVj7UumyrLGF",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 49,
+          "referenced_widgets": [
+            "cbda63c2d2de449baff69851ca9342f4",
+            "284ae90b84c040bdb227127b3428b471",
+            "3cc0f506cc8540e3a1024d3dc12d8dae"
+          ]
+        },
+        "cellView": "form",
+        "outputId": "3dc96e24-4aca-499b-b3fc-d386c92f8bc5"
+      },
+      "outputs": [
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "FileUpload(value={}, accept='.pdb', description='Upload')"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "cbda63c2d2de449baff69851ca9342f4"
+            }
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "# @title PDB file upload\n",
+        "# @markdown If your model needs a PDB file, run this cell and upload the file with the provided button that appears below.\n",
+        "# @markdown\n",
+        "# @markdown If you would like to change the file, simply upload another one. The last uploaded file will be what is used.\n",
+        "# @markdown If you would like to predict with the pre-loaded GB1 model, download (this pdb file)[https://github.com/gitter-lab/metl-pretrained/blob/main/pdbs/2qmt_p.pdb]\n",
```

Comment:
Do links not work in this Markdown syntax?

Reply:
When you put the brackets the right way round they do 😂